pytab/core.py

```diff
@@ -361,6 +361,21 @@ def cli(
     click.echo("| Obtaining System Information...", nl=False)
     system_info = hwi.get_system_info()
     click.echo(" success!")
+    click.echo("| Detected System Config:")
+    click.echo(f"|   OS: {system_info['os']['pretty_name']}")
+    for cpu in system_info['cpu']:
+        click.echo(f"|   CPU: {cpu['product']}")
+        click.echo(f"|     Threads: {cpu['cores']}")
+        click.echo(f"|     Arch: {cpu['architecture']}")
+    
+    click.echo("|   RAM:")
+    for ram in system_info['memory']:
+        click.echo(f"|     {ram['id']}: {ram['vendor']} {ram['size']} {ram['units']}")
```

Comment:
Make sure to only access fields where we tested before that they are present.

I marked this specifically here, because this was where the Script crashed on me. However this needs to be taken into account on every field, that is not strictly required for the script to run.

Fields like `os.pretty_name` should :tm: always exist though. (Cant realy hurt to check for their existance too though)